src/main/resources/application.properties.EXAMPLE

```diff
@@ -426,5 +426,8 @@ spring.devtools.restart.enabled=false
 ## Study Tag functionality
 #enable_study_tags=true|false
 
+## Custom Buttons
+# download_custom_buttons_json=classpath:/custom_buttons/download_custom_button_avm.json
```

Comment:
left resource path as "custom_buttons" in case other types are added later (less clutter with multiple folders)

Reply:
@pappde the example should be commented out so that when people copy this example file, they don't inadvertantly ...

Reply:
@alisman Yes, off by default, but easy to enable, and also on in the public instance.  For this reason, the **application.properties.EXAMPLE** has the the following line:

```
# download_custom_buttons_json=classpath:/custom_buttons/download_custom_button_avm.json
```

It is commented out here, but:

- Private Instance: can easily be enabled by copying and removing the #
- Public Instance: the line (uncommented) would be added to the application.properties

Reply:
ah.  i misread this line, not seeing the hash above it (it's wrapping in review interface)